pkg/plugins/golang/v3/scaffolds/init.go

```diff
@@ -34,7 +34,7 @@ import (
 
 const (
 	// ControllerRuntimeVersion is the kubernetes-sigs/controller-runtime version to be used in the project
-	ControllerRuntimeVersion = "v0.14.1"
+	ControllerRuntimeVersion = "v0.14.2"
 	// ControllerToolsVersion is the kubernetes-sigs/controller-tools version to be used in the project
 	ControllerToolsVersion = "v0.11.1"
```

Comment:
Since we haven't merged this PR yet, and controller-tools has a v0.11.3 release (https://github.com/kubernetes-sigs/controller-tools/blob/v0.11.3/go.mod#L16) that uses k8s 1.26.1, it would be nice to bump it here too. We can do it in a follow up as well, but would be helpful to track it with a comment or an issue.

Reply:
@varshaprasad96, I think we can do the bump in a follow up wdyt?

Reply:
Sounds good!

Reply:
I'll go ahead and merge this!